features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/recorder/base64/Base64Serializer.kt

```diff
@@ -27,7 +28,8 @@ class Base64Serializer private constructor(
     private val threadPoolExecutor: ExecutorService,
     private val drawableUtils: DrawableUtils = DrawableUtils(),
     private val base64Utils: Base64Utils = Base64Utils(),
-    private val webPImageCompression: ImageCompression = WebPImageCompression()
+    private val webPImageCompression: ImageCompression = WebPImageCompression(),
+    private val base64LruCache: Cache = Base64LRUCache
```

Comment:
I guess default parameters are not needed here? because this is a private constructor which is only invoked from `Base64Serializer.Builder.build` with all the necessary parameters provided, it is not invoked from the tests.